### PR TITLE
Improve test & output verification tasks

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject for less build time.

### CODE REVIEW

File: ballerina/build.gradle [hunk 1]
```diff
@@ -974,354 +1030,215 @@ task testExamples() {
...
             }
         }
+        
+        boolean buildFailure = false
+        File reportDetails = new File("${baseBuildPath}/result.txt")
+        if (!failedBuildExamples.isEmpty()) {
+            println "\n\tThe following BBEs failed when building\n"
+            failedBuildExamples.each {
+                println "\t ${it}"
+                reportDetails << "Build failed: ${it} \n"
```
Comment (ramithaYO) [inline ballerina/build.gradle:1225]: IMO, we don't need this because we print the failed BBEs'. WDYT?

**Review by ramithaYO — APPROVED** [review]

File: ballerina/build.gradle (hunk 1 above)
Comment (ramithaYO) [inline ballerina/build.gradle:1225]: IMO, we don't need this because we print the failed BBEs'. WDYT?

Comment (ramithaYO) [inline ballerina/build.gradle:1225]: IMO, we don't need this because we print the failed BBEs'. WDYT?

Comment (ramithaYO) [inline ballerina/build.gradle:1225]: IMO, we don't need to write the result to a file because we print the failed BBEs'. WDYT?

Comment (niveathika) [reply ballerina/build.gradle]: +1 fixed in 78aa2bc

**Pull request merged**